nix/mobile/android/android-env.nix

```diff
@@ -10,18 +10,18 @@ let
 
   androidComposition = androidenv.composeAndroidPackages {
     toolsVersion = "26.1.1";
-    platformToolsVersion = "28.0.2";
-    buildToolsVersions = [ "28.0.3" ];
+    platformToolsVersion = "29.0.5";
+    buildToolsVersions = [ "29.0.2" ];
```

Comment:
I wonder if we should try to read these values from `android/gradle.properties`

Reply:
Not sure how difficult that would be in Nix.

Reply:
I'll do it in another PR, since this fixes iOS fastlane build errors and I want to get this merged.